### ISSUE TITLE
ToggleSwitch height

### ISSUE THREAD

**Issue by fabads** [PramodJoshi/toggle_switch#3]
Hello,

Is there any way to control the height of the toggle switch ? I would like to set the same height as for other action chips I have also on the screen.

**Comment by PramodJoshi** [comment]
@fabads Yes, you can simply place `ToggleSwitch` in a `Container` and give it a height as shown below,

```dart
Container(
                  height: 100.0,
                  child: ToggleSwitch(
                    minWidth: 90.0,
                    initialLabelIndex: 2,
                    activeBgColor: Colors.redAccent,
                    activeTextColor: Colors.white,
                    inactiveBgColor: Colors.grey,
                    inactiveTextColor: Colors.grey[900],
                    labels: ['America', 'Canada', 'Mexico'],
                    onToggle: (index) {
                      print('switched to: $index');
                    },
                  )
              )
```

**Screenshot:**

![screenshot](https://user-images.githubusercontent.com/17320471/85310850-218b2b80-b47a-11ea-9ec6-5fb7d116a699.png)

**Comment by fabads** [comment]
Thank you very much @PramodJoshi for the hint, it works fine (in my case, wanted to reduce the size).

**Comment by PramodJoshi** [comment]
FYI, `minHeight` is not available in `toggle_switch: "^0.1.5"`. Cheers!
